app/assets/javascripts/dradis/plugins/csv/upload.js

```diff
@@ -15,24 +15,58 @@ window.addEventListener('job-done', function(e){
 
 document.addEventListener('turbolinks:load', function() {
   if ($('body.upload.new').length) {
-    $('[data-behavior=type-select]').on('change', function() {
+    $('[data-behavior=type-select]').on('change', function(e) {
       var $nodeSelect = $('select option[value="node"]:selected').parent();
 
+      // Disable Node Label option
       $('[data-behavior=type-select]').each(function(i, select) {
         var $tr = $(select).closest('tr');
-
-        $tr.find('[data-behavior=na-field-label]').addClass('d-none');
-        $tr.find('[data-behavior=default-field-label]').removeClass('d-none');
-
         if ($nodeSelect.length && !$nodeSelect.is($(select))) {
           $(select).find('option[value="node"]').attr('disabled', 'disabled');
         } else {
           $(select).find('option[value="node"]').removeAttr('disabled');
         }
       });
 
-      $nodeSelect.closest('tr').find('[data-behavior=na-field-label]').removeClass('d-none');
-      $nodeSelect.closest('tr').find('[data-behavior=default-field-label]').addClass('d-none');
+      // Update fields column labels
+      var hasNoFields = $(e.target).val() == 'skip' || $(e.target).val() == 'node';
+      if (hasNoFields) {
+        $(e.target).closest('tr').find('[data-behavior=field-label]').text('N/A');
+      }
+      else {
+        var header = $(e.target).closest('tr').find('td:nth-child(2)').text();
+        $(e.target).closest('tr').find('[data-behavior=field-label]').text(header);
+      }
+
+
+      setDradisFieldSelect($(e.target));
+    });
+
+    $('[data-behavior=identifier]').on('mousedown', function(e){
```

Comment:
```javascript
    $('[data-behavior=identifier]').on('change', function(e) {
      var $tr = $(e.target).closest('tr');
      var $fieldSelect = $tr.find('[data-behavior=field-select]'),
          $typeSelect = $tr.find('[data-behavior=type-select]');

      $fieldSelect.attr('disabled', 'disabled');
      // With RTP
      $fieldSelect.html($('<option selected></option>').attr('value', 'plugin_id').text('plugin_id'));
      // With no RTP
      $tr.find('[data-behavior=field-label]').text('plugin_id');

      $typeSelect.attr('disabled', 'disabled');
      $typeSelect.val('issue').change();

      var $trs = $('select option[value="issue"]:selected').closest('tr').not($tr);
      $trs.find('[data-behavior=field-select]').removeAttr('disabled');
      $trs.find('[data-behavior=type-select]').removeAttr('disabled');

      $trs.each(function(i, tr) {
        var $tr = $(tr);
        setDradisFieldSelect($tr.find('[data-behavior~=type-select]'));

        // With no RTP
        var header = $tr.find('td:nth-child(2)').text();
        $tr.find('[data-behavior=field-label]').text(header);
      })
    })
```

Reply:
That bug was just a matter of ordering things. Fixed in the recent commit